flask_restless/helpers.py

```diff
@@ -622,3 +640,36 @@ def __call__(self, model, _apimanager=None, **kw):
 #:     <class 'mymodels.Person'>
 #:
 model_for = ModelFinder()
+
+#: Returns the primary key to be used for the given model, as specified
```

Comment:
The documentation needs to be changed to indicate now that the argument can be either a model class or an instance of the model.